packages/indexer-service/src/allocations.ts

```diff
@@ -109,18 +110,23 @@ export interface EnsureAttestationSignersOptions {
   logger: Logger
   allocations: Eventual<Allocation[]>
   wallet: Wallet
+  chainId: number
+  disputeManagerAddress: string
 }
 
-export type AttestationSignerCache = LRUCache<string, string>
-export type AttestationSignerMap = Map<string, string>
+export type AttestationSignerCache = LRUCache<string, NativeAttestationSigner>
+export type AttestationSignerMap = Map<string, NativeAttestationSigner>
```

Comment:
What a neat way to slot these in :)

Reply:
Yeah, I kind of lucked out that you already had all the infrastructure I needed written already.